README.md

```diff
@@ -1,31 +1,65 @@
-docker base
-===========
+# docker base
 
 This role installs baseline needs for machines running Docker, beyond the Docker Engine itself,
 including:
 
  -  Requirements for running Ansible's `docker` module
- -  Requirements for running Docker containers via `upstart` (optional)
  -  Utilities for garbage collecting unused Docker resources
  -  Customized configurations of the Docker Engine
- -  Installs docker-compose
+ -  Installs docker-compose (optional)
  -  Pulls some common images that we anticipate using in many circumstances
 
-Role Variables
---------------
+## Role Variables
 
 This role expects the following variables:
 
- -  `docker_log_rotate_interval` controls the frequency of Docker log rotation
- -  `docker_log_rotate_count` controls the number of Docker logs to retain
- -  `docker_uses_upstart` defines whether `upstart` support will be installed
- -  `docker_py_version` defines the version of the Docker Python library being used
- -  `docker_compose_version` defines the version of docker-compose to install (defaults to newest)
+ - `docker_version`: the version of Docker that is installed on the target host. Note that docker
```

Comment:
By more straightforward, I mean:

- more loosely coupled / less assumptions about other roles
- one less parameter / less documentation
- no more chance of a mismatch between docker_version and the actual docker version